### PR TITLE
MTROPOLIS: Add detection for Muppet Treasure Island (EN, IT, ES (MX), EN demo)

### PR DESCRIPTION
**Pull request by einstein95**
Demo: https://drive.google.com/file/d/1zeVS-HiI-sUrsKnuaqW3__UK_gSOJxVj/view?usp=sharing

### CODE REVIEW

File: engines/mtropolis/detection_tables.h [hunk 1]
```diff
@@ -329,6 +330,27 @@ static const MTropolisGameDescription gameDescriptions[] = {
...
+			{
+				{ "MTIWIN95.EXE", 0, "aad51b462d0961fb02d9c1422a41937f", 840192 },
+				{ "GROUP3.R95",	  0, "3b01850e511727aa270aff1d6cb1fcf8", 89088 },
+				{ "MTIKIT.R95",	  0, "f7183d9ff845a3a607f764920fc23b18", 101376 },
+				{ "MUP_DATA.MPL", 0, "aea8ca15455991278213d09674a183ed", 51678610 },
+				AD_LISTEND
+			},
+			Common::EN_ANY,
+			Common::kPlatformWindows,
+			ADGF_DEMO | ADGF_TESTING,
```
Comment (bluegr) [inline engines/mtropolis/detection_tables.h:346]: Is this in a good state to be tested?

Comment (einstein95) [reply engines/mtropolis/detection_tables.h:346]: Ah whoops, copy/paste error

Comment (bluegr) [reply engines/mtropolis/detection_tables.h:346]:
```suggestion
			ADGF_DEMO,
```

**Comment by einstein95** [comment]
It'll need to be squashed before merging, I've just been working via browser while doing Windows expletives.

**Pull request merged**